_source/_posts/2018-05-18-node-authentication-with-passport-and-oidc.md

```diff
@@ -127,43 +141,41 @@ app.use(passport.session());
+Now you'll configure Passport.js to use Okta as your Identity Provider (IdP). To do this, tell passport to use the `Strategy` created in the requirements, right below the Passport.js configuration from the last section.
 
 ```js
 // set up passport
-passport.use('oidc', new OidcStrategy({
-  issuer: 'https://{yourOktaDomain}/oauth2/default',
-  authorizationURL: 'https://{yourOktaDomain}/oauth2/default/v1/authorize',
-  tokenURL: 'https://{yourOktaDomain}/oauth2/default/v1/token',
-  userInfoURL: 'https://{yourOktaDomain}/oauth2/default/v1/userinfo',
-  clientID: '{ClientID}',
-  clientSecret: '{ClientSecret}',
+passport.use('oidc', new Strategy({
+  issuer: '{OKTA_OAUTH2_ISSUER}',
+  authorizationURL: 'https://{OKTA_DOMAIN}/oauth2/default/v1/authorize',
+  tokenURL: 'https://{OKTA_DOMAIN}/oauth2/default/v1/token',
+  userInfoURL: 'https://{OKTA_DOMAIN}/oauth2/default/v1/userinfo',
+  clientID: '{OKTA_OAUTH2_CLIENT_ID}',
+  clientSecret: '{OKTA_OAUTH2_CLIENT_SECRET}',
   callbackURL: 'http://localhost:3000/authorization-code/callback',
   scope: 'openid profile'
-}, (issuer, sub, profile, accessToken, refreshToken, done) => {
+}, (issuer, profile, done) => {
   return done(null, profile);
 }));
 ```
 
-The code above sets the name of the strategy as 'oidc' and set all the URLs that the strategy needs to know to pull off the authorization code flow for OpenID Connect. The issuer is the URL for your authorization server that was created for you when you signed up for an Okta developer account. You can view it by clicking on **API** in your Okta dashboard and choosing the **Authorization Servers** tab. To find the `authorizationURL`, `tokenURL` and `userInfoURL` settings, you can click on the default authorization server and view its settings. There is a **Metadata URI** setting that, by clicking on the link, will show you the `.well-known` document. This document tells anyone using this authorization server about the information and endpoints it can provide.
+Be sure to replace the placeholder variables with your actual Okta information.
+
+- Replace `{OKTA_OAUTH2_ISSUER}` with your Org's OAuth2 Issuer URL.
+- Replace `{OKTA_OAUTH2_CLIENT_ID}` with the Client ID of your application.
+- Replace `{OKTA_OAUTH2_CLIENT_SECRET}` with the Client secret of your application.
+- Replace `{OKTA_DOMAIN}` with your Org's Okta domain. It is the same as your OAuth2 Issuer URL without the `https://` and `/oauth2/default` segments.
 
-{% img blog/node-passport/default-authorization-server-settings.png alt:"The default authorization server settings" width:"800" %}{: .center-image }
+The code above sets the name of the strategy as 'oidc' and set all the URLs that the strategy needs to know to pull off the authorization code flow for OpenID Connect. 
```

Comment:
```suggestion
The code above sets the name of the strategy as 'oidc.' Then it sets all the URLs that the strategy needs to complete the authorization code flow for OpenID Connect. 
```
Two sentences to route around the complexity. Maybe you don't even need a second verb: "Then it sets all the URLs that the strategy needs for the OpenID Connect authorization code." - but maybe this one is too terse. Your call. :-)